starter/source/elements/sh3n/coquedk/cmaini3.F

```diff
@@ -89,7 +92,7 @@ SUBROUTINE CMAINI3(ELBUF_STR,PM      ,GEO      ,NEL      ,NLAY      ,
 C-----------------------------------------------
       INTEGER IT,IL,NUVAR,NPTT,ILAW
       my_real
-     .   VX(MVSIZ),VY(MVSIZ),VZ(MVSIZ),PHI1(NLAY,MVSIZ),PHI2(NLAY,MVSIZ)
+     .   VX(MVSIZ),VY(MVSIZ),VZ(MVSIZ),PHI1(NPT_ALL,MVSIZ),PHI2(NPT_ALL,MVSIZ)
```

Comment:
What is the a typical value for NPT_ALL? I am worried about the stack size here.